src/sage/functions/orthogonal_polys.py

```diff
@@ -2246,62 +2271,65 @@ class Func_ultraspherical(GinacFunction):
 
     EXAMPLES::
 
-        sage: ultraspherical(8, 101/11, x)
+        sage: ultraspherical(8, 101/11, x)                                              # needs sage.symbolic
         795972057547264/214358881*x^8 - 62604543852032/19487171*x^6...
         sage: x = PolynomialRing(QQ, 'x').gen()
-        sage: ultraspherical(2,3/2,x)
+        sage: ultraspherical(2, 3/2, x)                                                 # needs sage.symbolic
         15/2*x^2 - 3/2
-        sage: ultraspherical(1,1,x)
+        sage: ultraspherical(1, 1, x)                                                   # needs sage.symbolic
         2*x
-        sage: t = PolynomialRing(RationalField(),"t").gen()
-        sage: gegenbauer(3,2,t)
+        sage: t = PolynomialRing(RationalField(), "t").gen()
+        sage: gegenbauer(3, 2, t)                                                       # needs sage.symbolic
         32*t^3 - 12*t
-        sage: x = SR.var('x')
+        sage: x = SR.var('x')                                                           # needs sage.symbolic
         sage: n = ZZ.random_element(5, 5001)
         sage: a = QQ.random_element().abs() + 5
-        sage: s = (  (n+1)*ultraspherical(n+1,a,x)
-        ....:      - 2*x*(n+a)*ultraspherical(n,a,x)
-        ....:      + (n+2*a-1)*ultraspherical(n-1,a,x) )
-        sage: s.expand().is_zero()
+        sage: s = (  (n + 1)*ultraspherical(n + 1, a, x)                                # needs sage.symbolic
+        ....:      - 2*x*(n + a)*ultraspherical(n, a, x)
+        ....:      + (n + 2*a - 1)*ultraspherical(n - 1, a, x) )
+        sage: s.expand().is_zero()                                                      # needs sage.symbolic
         True
-        sage: ultraspherical(5,9/10,3.1416)
+        sage: ultraspherical(5, 9/10, 3.1416)                                           # needs sage.symbolic
         6949.55439044240
-        sage: ultraspherical(5,9/10,RealField(100)(pi))
+        sage: ultraspherical(5, 9/10, RealField(100)(pi))                               # needs sage.rings.real_mpfr
         6949.4695419382702451843080687
```

Comment:
block-scoped, separating the last line?

Reply:
You may prefer not separating a single doctest. Then just let me know.

Reply:
actually also the last line of this block needs `sage.symbolic`

Reply:
Then would you fix?

Reply:
It is fixed by adding the block tag

Reply:
Yes. I confirm that. 

Strangely, github does not show the last changes...

Reply:
Now I see the changes after setting "Changes from all commits". 

OK.